src/base.jl

```diff
@@ -1184,16 +1167,10 @@ function get_components_by_name(
     return IS.get_components_by_name(T, sys.data, name)
 end
 
-"""
```

Comment:
Moved to `get_components_interface` (though there's an argument it should appear here too)